rate-providers/LoopRateProvider.md

```diff
@@ -0,0 +1,53 @@
+# Rate Provider: LoopRateProvider
+
+## Details
+- Reviewed by: @brunoguerios
+- Checked by: @\<GitHub handle of secondary reviewer\>
+- Deployed at:
+    - [ethereum:0x1f037c849CF2448d67A120543EA4ec3CE5A95FcA](https://etherscan.io/address/0x1f037c849CF2448d67A120543EA4ec3CE5A95FcA)
+    - [\<audit title\>](\<link to audit\>)
+
+## Context
+\<Write a brief description of the intended functionality here.\>
+
+## Review Checklist: Bare Minimum Compatibility
+Each of the items below represents an absolute requirement for the Rate Provider. If any of these is unchecked, the Rate Provider is unfit to use.
+
+- [x] Implements the [`IRateProvider`](https://github.com/balancer/balancer-v2-monorepo/blob/bc3b3fee6e13e01d2efe610ed8118fdb74dfc1f2/pkg/interfaces/contracts/pool-utils/IRateProvider.sol) interface.
+- [x] `getRate` returns an 18-decimal fixed point number (i.e., 1 == 1e18) regardless of underlying token decimals.
+
+## Review Checklist: Common Findings
+Each of the items below represents a common red flag found in Rate Provider contracts.
+
+If none of these is checked, then this might be a pretty great Rate Provider! If any of these is checked, we must thoroughly elaborate on the conditions that lead to the potential issue. Decision points are not binary; a Rate Provider can be safe despite these boxes being checked. A check simply indicates that thorough vetting is required in a specific area, and this vetting should be used to inform a holistic analysis of the Rate Provider.
+
+### Administrative Privileges
+- [ ] The Rate Provider is upgradeable (e.g., via a proxy architecture or an `onlyOwner` function that updates the price source address).
+
+- [x] Some other portion of the price pipeline is upgradeable (e.g., the token itself, an oracle, or some piece of a larger system that tracks the price).
+    - upgradeable component: `PoolV3` ([ethereum:0xa684EAf215ad323452e2B2bF6F817d4aa5C116ab](https://etherscan.io/address/0xa684EAf215ad323452e2B2bF6F817d4aa5C116ab))
+    - admin address: [ethereum:0x9613E12A424B4CbaCF561F0ec54b418c76d6B26D](https://etherscan.io/address/0x9613E12A424B4CbaCF561F0ec54b418c76d6B26D)
+    - admin type: EOA
```

Comment:
PoolV3 has an `addressProvider`, which is the EOA that acts as owner/controller, but the actual component that can be changed (and effectively affect the rate) is another contract called `interestRateModel`. Let me know if I should fill out the interestRateModel contract/address here instead.

Reply:
Synced with MkFlow that we should only care about upgradeable contracts that are relevant to the getRate flow, more specifically, that can manipulate the rate. This is not the case for the comment above, so I marked it as not upgradeable.